src/main/kotlin/com/tradeshift/blayze/Model.kt

```diff
@@ -95,25 +97,25 @@ class Model(
      *
      * @return predicted outcomes and their probability, e.g. {"positive": 0.3124, "negative": 0.6876}
      */
-    fun predict(inputs: Inputs): Map<Outcome, Double> {
+    fun predict(inputs: Inputs, pseudoCounts: Map<FeatureName, PseudoCount>): Map<Outcome, Double> {
```

Comment:
can you make a function with the same signature that uses the default values.

so this would work `predict(inputs)`